modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/search/index/TaskWorkflowMetricsIndexerImpl.java

```diff
@@ -307,26 +309,38 @@ public Document updateTask(
 
 				ScriptBuilder builder = scripts.builder();
 
+				builder.idOrCode(
+					StringUtil.read(
+						getClass(),
+						"dependencies/workflow-metrics-update-task-" +
+							"script.painless")
+				).language(
+					"painless"
+				).putParameter(
+					"assigneeIds", assigneeIds
+				).putParameter(
+					"assigneeType", assigneeType
+				);
+
+				if (Objects.equals(assigneeType, User.class.getName()) &&
```

Comment:
Hey @MarinhoFeliphe, put the if statement after `"assigneeIds", assigneeIds`(alphabetical order). please.